### PR TITLE
docs(graphql): added docs for graphql model shim

### PR DESCRIPTION
**Pull request by maayanbar13**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x] Docs
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

Added another doc page in the `graphql` section to document how to share `Code-first` `graphql` models with your front-end application.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

Documents the model shim feature from [nestjs/graphql#1184](https://github.com/nestjs/graphql/pull/1184).

### CODE REVIEW

File: content/graphql/sharing-models.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+### Sharing Models
+
+> warning **Warning** This chapter applies only to the code first approach.
+
+One of the biggest advantages of using Typescript for the backend of your project is the ability to reuse the same models in a Typescript-based frontend application, by using a common Typescript package.    
+
+but there's a problem: the models created using the Code first approach are heavily decorated with graphql related decorators. Those decorators are irrelevant in the frontend, negatively impacting performance.
```
Comment (johannesschobel) [inline content/graphql/sharing-models.md:7]:
```suggestion
But there's a problem: the models created using the code first approach are heavily decorated with graphql related decorators. Those decorators are irrelevant in the frontend, negatively impacting performance.

```

File: content/graphql/sharing-models.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+> warning **Warning** This chapter applies only to the code first approach.
+
+One of the biggest advantages of using Typescript for the backend of your project is the ability to reuse the same models in a Typescript-based frontend application, by using a common Typescript package.    
+
+but there's a problem: the models created using the Code first approach are heavily decorated with graphql related decorators. Those decorators are irrelevant in the frontend, negatively impacting performance.
+
+#### Using the model shim
+
+To solve this issue, nest provides a "shim" which allows you to replace the original decorators with inert code by using a `webpack` (or similar) configuration.
```
Comment (johannesschobel) [inline content/graphql/sharing-models.md:11]:
```suggestion
To solve this issue, nestjs provides a "shim" which allows you to replace the original decorators with inert code by using a `webpack` (or similar) configuration.    

```

Comment (jmcdo29) [reply content/graphql/sharing-models.md:11]: I'd say just capitalize "Nest" here. 

```suggestion
To solve this issue, Nest provides a "shim" which allows you to replace the original decorators with inert code by using a `webpack` (or similar) configuration. 
```

File: content/graphql/sharing-models.md (hunk 1 above)
Comment (jmcdo29) [reply content/graphql/sharing-models.md:7]:
```suggestion
But there's a problem: the models created using the code first approach are heavily decorated with GraphQL related decorators. Those decorators are irrelevant in the frontend, negatively impacting performance.
```
```

**Review by jmcdo29 — CHANGES_REQUESTED** [review]
Nice addition! I added a few comments.

**Review by jmcdo29 — APPROVED** [review]

**Comment by maayanbar13** [comment]
Any reason this is still blocked?

**Review by micalevisk — APPROVED** [review]

**Review by Tony133 — APPROVED** [review]

File: content/graphql/sharing-models.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+### Sharing Models
```
Comment (kamilmysliwiec) [inline content/graphql/sharing-models.md:1]:
```suggestion
### Sharing Models

```

File: content/graphql/sharing-models.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
```
Comment (kamilmysliwiec) [inline content/graphql/sharing-models.md:1]:
```suggestion
### Sharing models

```

**Pull request merged**